dev-scheme/chicken/chicken-5.3.0.recipe

```diff
@@ -55,6 +55,7 @@ BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	"
 BUILD_PREREQUIRES="
+	cmd:chicken
```

Comment:
Seems chicken requires chicken to build now, if this one is OK it should be good to merge